kube-derive/src/custom_resource.rs

```diff
@@ -192,6 +198,109 @@ impl FromMeta for SchemaMode {
     }
 }
 
+/// This struct mirrors the fields of `k8s_openapi::CustomResourceSubresourceScale` to support
+/// parsing from the `#[kube]` attribute.
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct Scale {
+    pub(crate) label_selector_path: Option<String>,
```

Comment:
Seems we can expose this also as it's part of the api now in https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource